config/prod/roche-ckan-awslinux

```diff
@@ -0,0 +1,35 @@
+# Provision an EC2 instance
+template_path: managed-ec2-v5.yaml
+stack_name: roche-ckan-awslinux
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "Platform"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Roche/GNE"
+  # The resource owner
+  OwnerEmail: "kimyen.ladia@sagebionetworks.org"
+
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.medium"
+  # (Optional) ID of the base AMI (default: ami-97785bed) (supported distros: AWS linux)
+  AMIId: "ami-97785bed"      
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  VolumeSize: "50"
+  # (Optional) Run EC2 in a subnet (default: PrivateSubnet) (valid values: PrivateSubnet or PublicSubnet)
+  VpcSubnet: "PublicSubnet"
```

Comment:
this line and the next will make your instance have a public IP address and expose port 80 on the instance.  Unless this is intentional we recommend removing these parameters so that the instance is private and does not expose any ports which would be much more secure.

Reply:
This is for testing, so I wanted it to be public. I planned to remove the instance after I'm done testing.